### DISCUSSION TITLE
Question on limitations of 'Table' component - JS.

### DISCUSSION THREAD

**Discussion by tokkisva** [Budibase/budibase#9019] [Help]
Hi All,

I'm trying to take a value from a column in a Table that is connected to a REST datasource, and do a lookup against a Budibase DB Datasource, then return a value from the Budibase DB when it finds a match.

![image](https://user-images.githubusercontent.com/38924262/207138029-8b2e7094-ad29-436f-9b15-6edc2169cc53.png)
Table config
![image](https://user-images.githubusercontent.com/38924262/207137822-427ce4f2-ce5f-4a6b-bc32-bdf90fb33cfd.png)

Example: If source has a value of **10**, look up value **10** in the Budibase DB rows on a specific column, pull a string from another column on the matching row, place that string in the column/row on the table.

I noticed that whether I use Handlebars or JS within a table and call things from another datasource (Configure Columns -> Column Settings -> Value -> {JScode}), the values just don't populate. When used in other components, the values do populate. 

Do you know if table values can be populated via JS from other datasources, or is there a limitation here I'm hitting?

Thanks in advance!

**Comment by melohagan** [comment]
Hey @tokkisva 

Could you share the JavaScript binding please? It'll hopefully help me understand the scenario better!
